staging/traefik-forward-auth/templates/deployment.yaml

```diff
@@ -105,13 +110,7 @@ spec:
                 secretKeyRef:
                   name: {{ template "traefik-forward-auth.fullname" . }}-secret
                   key: secret
-            {{- if .Values.traefikForwardAuth.allowedUser.valueFrom.secretKeyRef }}
-            - name: WHITELIST
-              valueFrom:
-                secretKeyRef:
-                  name: {{ required "traefikForwardAuth.allowedUser.valueFrom.secretKeyRef.name is required" .Values.traefikForwardAuth.allowedUser.valueFrom.secretKeyRef.name }}
-                  key: {{ required "traefikForwardAuth.allowedUser.valueFrom.secretKeyRef.key is required" .Values.traefikForwardAuth.allowedUser.valueFrom.secretKeyRef.key }}
-            {{- else if gt (len .Values.traefikForwardAuth.whitelist) 0 }}
+            {{- if gt (len .Values.traefikForwardAuth.whitelist) 0 }}
```

Comment:
os if secretKeyRef is not defined, the whitelist will not be defined either? before there was an else if, is this what you intend?

Reply:
yes, the ops-portal user no longer needs to be added to the whitelist since the user is also bound to the cluster-admin role and will be permitted by the default access policy.